torch/_higher_order_ops/scan.py

```diff
@@ -48,6 +48,14 @@ def _extract_carry_and_out(flat_out: List[Any], num_carry: int):
     return flat_out[:num_carry], flat_out[num_carry:]
 
 
+# An empty function that's used for inductor lowering in lowering.py
+# Compared with aten.select, the lowring rule is more specialized
+# to the scan operator. For example, we skips a bunch of checks that we're
+# ceratin to be true for scan.
+def scan_slice_view(dst, dim, idx):
+    pass
```

Comment:
yean, sounds good! I can change it to `unsafe_select`.

A hop need to inherit from HigherOrderOp so this is just a normal function. I put into the scan file because it's kind of specific to scan lol.